helm/hpcc/templates/_helpers.tpl

```diff
@@ -73,13 +73,13 @@ esp:
 secretTimeout: {{ .Values.secrets.timeout | default 300 }}
 storage:
   daliPlane: {{ $daliStoragePlane }}
-  dllsPlane: {{ $daliStoragePlane }}
+  dllsPlane: {{ $dllStoragePlane }}
```

Comment:
@gavin btw - this was clearly wrong before, but nothing used global.storage.dllsPlane so ..